com.termius.termius.yml

```
@@ -0,0 +1,73 @@
+app-id: com.termius.termius
+
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: "18.08"
+
+command: termius-app
+
+finish-args:
+  # X11/Wayland
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  # Sync
+  - --share=network
+  # Filesystem access
+  - --filesystem=host
+  # Allow access to keyring
+  - --talk-name=org.freedesktop.secrets
+  # SSH
+  - --socket=ssh-auth
+  - --own-name=org.KDE.*
+
+modules:
+
+  - name: termius
+    buildsystem: simple
+    build-commands:
+        - install -Dm755 termius-app.sh /app/bin/termius-app
+        - install -Dm755 apply_extra.sh /app/bin/apply_extra
+        - install -Dm644 ${FLATPAK_ID}.appdata.xml -t /app/share/appdata/
```

```suggestion
        - install -Dm644 ${FLATPAK_ID}.appdata.xml -t /app/share/metainfo/
```
Appdata is the old install location.